scripts/herodotus-interaction-demo.ts

```diff
@@ -212,7 +239,9 @@ async function main() {
           {
             index: '0x0',
             params: CallData.compile({
-              storageProof: storageProofLittleEndianWords64,
+              checkpoint_index: numCheckpoints - BigInt(1),
```

Comment:
it would be zero if they have not delegated their tokens to themself yet. In which case the vp is zero. So would just need a check in sx.js to ensure the voter does indeed have before this call is made (which would just fail otherwise)